### ISSUE TITLE
Option to keep multiple comments

### ISSUE THREAD

**Issue by DAmNRelentless** [tamasfe/taplo#137]
Before formatting my file looks like this:
![](https://i.imgur.com/Bv4qnRy.png)

After formatting, it looks like this:
![](https://i.imgur.com/3n6Z7ay.png)

Looks like the formatting is moving the comment behind the previous key when there are multiple comment lines.
This is a problem since both lines describe the next key in the TOML.
It would be nice to have an option to keep multi-comments.

Extension Version: 0.13.1

VSCode
Version: 1.58.0 (user setup)
Commit: 2d23c42a936db1c7b3b06f918cde29561cc47cd6
Date: 2021-07-08T06:54:55.083Z
Electron: 12.0.13
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.19042

**Comment by tamasfe** [comment]
Yes, it's definitiely a bug in the new formatter, sorry.

Can you please provide code blocks instead of screenshots? Would help a lot with just copy/pasting the expected behaviour in a test.

**Comment by DAmNRelentless** [comment]
sure thing, this is enough to reproduce it
```toml
#General settings
[general]
	#Is Enabled?
	enabled = true
	#Cost
	#Range: > -2147483648
	cost = 10
	#Is Starter Glyph?
	starter = false
	#The maximum number of times this glyph may appear in a single spell
	#Range: > 1
	per_spell_limit = 2147483647
```

**Comment by tamasfe** [comment]
Thanks!

Should be fixed in 2d87495d3e04a79adb1cc9346db8d63c640a7509, I'll promptly push a new release with it.

**Comment by DAmNRelentless** [comment]
Thanks for the fast response and the fast work! :)
